servicetalk-http-api/src/test/java/io/servicetalk/http/api/DefaultHttpExecutionStrategyTest.java

```diff
@@ -253,59 +227,64 @@ StreamingHttpResponse createNewResponse() {
         Publisher<Object> instrumentedFlatRequestForClient(Publisher<Object> req) {
             return req.beforeRequest(__ -> {
                 analyzed.set(SEND_ANALYZED_INDEX, true);
-                verifyThread(offloadSend, "Unexpected thread requested from request.");
+                verifyThread(strategy.isSendOffloaded(), "Unexpected thread requested from request.");
             });
         }
 
         <T> Single<T> instrumentSend(Single<T> original) {
+            return original.beforeOnSuccess((T t) -> {
+                analyzed.set(SEND_ANALYZED_INDEX, true);
+                verifyThread(strategy.isSendOffloaded(), "Unexpected thread requested from success.");
+            });
+        }
+
+        <T> Single<T> instrumentSendCancel(Single<T> original) {
```

Comment:
If this test still causes issues, let's disable it for now to unblock other PRs.